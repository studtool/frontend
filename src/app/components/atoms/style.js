export const colors = {
    primary : '#5952de',
    primaryLight: '#917fff',
    primaryDark: '#0928ab',
    secondary: '#f44336',
    secondaryLight: '#ff7961',
    secondaryDark: '#ba000d',
    textOnPrimary: '#ffffff',
    textOnSecondary: '#000000',
    black: '#000000',
    gray: '#757575',
    white: '#ffffff',
}
