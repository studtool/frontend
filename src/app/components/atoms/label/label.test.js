import React from 'react';
import {Label} from './label';
import {colors} from '../style';
import renderer from 'react-test-renderer'
import 'jest-styled-components'

test('label test', () => {
    const tree = renderer.create(<Label/>).toJSON();
    expect(tree).toMatchSnapshot();
});
