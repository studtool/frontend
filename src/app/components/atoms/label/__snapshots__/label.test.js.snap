// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`label test 1`] = `
<label
  className=""
/>
`;
